plugins/modules/swim_workflow_manager.py

```diff
@@ -34,6 +34,16 @@
     description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
     type: bool
     default: False
+  max_timeout:
+    description: Sets the maximum duration in seconds for attempting to retrieve task details from the API. If the
+        task details are not obtained within this timeframe, the loop will be terminated, and a log message will be
+        generated indicating the timeout.
+    type: int
+    default: 1200
+  interval_seconds:
```

Comment:
Can we say like
dnac_task_poll_interval or dnac_ api_polling_interval and write documentation in plugins/doc_fragments/workflow_manager_params.py and intent_params.py?

Reply:
added.